cocos/platform/ios/CCImage-ios.mm

```diff
@@ -91,7 +91,14 @@ of this software and associated documentation files (the "Software"), to deal
     CGBitmapInfo bitmapInfo = kCGBitmapByteOrderDefault;
     if (saveToPNG && hasAlpha() && (! isToRGB))
     {
-        bitmapInfo |= kCGImageAlphaPremultipliedLast;
+		if (_hasPremultipliedAlpha)
+		{
+			bitmapInfo |= kCGImageAlphaPremultipliedLast;
+		}
+		else
+		{
+			bitmapInfo |= kCGImageAlphaLast;
+		}
```

Comment:
could you please fix the indent

Reply:
Sorry I didn't notice the .mm files not having correct formatting in my editor. Converted the tabs to spaces, so the indentation is fixed now.